woocommerce/woocommerce
# [3.6] Improve appearance of no-suggestions state in product edit metabox

### Changes proposed in this Pull Request:
Adds bottom margin so this doesn't hard-align when metabox is small. Reduced top margin to keep this element compact.

### How to test the changes in this Pull Request:
1. Get this source code running as your woocommerce plugin in a test site.
2. Edit a product. 
3. Select `Grouped product`.
4. Click `Get more options` tab and dismiss all suggestions. No-suggestions state should like reasonable and nice.

<img width="860" alt="Screen Shot 2019-04-05 at 4 39 16 PM" src="https://user-images.githubusercontent.com/4167300/55602714-73875900-57c3-11e9-9c72-14538cf07f47.png">
